### PR TITLE
add qhui-red caption files

### PR DESCRIPTION
**Pull request by qhui-red**
# h1

##### h5

### CODE REVIEW

**Comment by salmonslap** [comment]
🧀

File: _posts/0000-11-01-qhui-red.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+![leia](https://cloud.githubusercontent.com/assets/16547949/25400918/17c5d2e4-29c2-11e7-92ef-79bacb424ef4.jpg)
+{: .center}
```
Comment (salmonslap) [inline _posts/0000-11-01-qhui-red.md:7]: Test Comment

**Review by salmonslap — APPROVED** [review]

**Review by qhui-red — COMMENTED** [review]
LOL

File: _posts/0000-11-01-qhui-red.md (hunk 1 above)
Comment (salmonslap) [reply _posts/0000-11-01-qhui-red.md:7]: <h1>Header in a comment!</h1>
<ul>
<h2>List!</h2>
<li>test</li>
<li><h3>Header In List!</h3></li>
</ul>

**Comment by shengzhang09** [comment]
:rofl:

**Pull request merged**